composer.json

```diff
@@ -68,19 +68,19 @@
     }
   },
   "scripts": {
-    "test": [
-      "@phpstan",
-      "@phpcs",
-      "@phpmd",
-      "@phpunit",
-      "@coverage"
+    "test:all": [
```

Comment:
Why not just leave this one as `test`?

Reply:
It's `test:all`, maybe outdated diff

Reply:
No, my point was that I don't think this should have changed; that `test` is a better option than `test:all`. But also I'm not so committed to this idea that it should prevent merging.

I'm really just looking for you to justify why `test:all` is a better keyword.